src/test/shell/bazel/remote/remote_execution_test.sh

```diff
@@ -1397,6 +1397,81 @@ EOF
   expect_log "Setting both --remote_default_platform_properties and --remote_default_exec_properties is not allowed"
 }
 
+function test_platform_default_properties_invalidation_with_platform_exec_properties() {
+  # Test that when changing values of --remote_default_platform_properties all actions are
+  # invalidated.
+  mkdir -p test
+  cat > test/BUILD << 'EOF'
+platform(
+    name = "platform_without_any_exec_properties",
```

Comment:
I'm confused because this platform **does** have exec properties.

Reply:
Copy-paste gone wrong. Cleaned up in 30709fdfd9